README.md

```diff
@@ -100,7 +100,53 @@ public String TrimString(String s) {
 ## Class members
 
 - Avoid `this.` unless absolutely necessary.
-- Always specify member visiblity, even if it's the default (i.e. `private string foo;` not `string _foo;`).
+
+DO
+```
+public class Node 
```

Comment:
Minor: I thought about
```
public UserController(IUserSettingsRepository userSettingsRepository)
{
    _userSettingsRepository = userSettingsRepository;
}
```
and

```
public UserController(IUserSettingsRepository userSettingsRepository)
{
    this._userSettingsRepository = userSettingsRepository;
}
```
to make it simplier and more related to our code :wink: But you sample works for me too :+1: